MSAL/src/native_auth/network/responses/validator/sign_in/validated_response/MSALNativeAuthSignInChallengeValidatedResponse.swift

```diff
@@ -39,6 +39,7 @@ enum MSALNativeAuthSignInChallengeValidatedErrorType: Error {
     case invalidServerResponse
     case userNotFound(message: String?)
     case unsupportedChallengeType(message: String?)
+    case unexpectedError(message: String?)
```

Comment:
for this response we already have an "invalidServerResponse" that kind of overlap this "unexpectedError". Consider merging these two validated error response

Reply:
Done, changed all to .unexpectedError as invalidServerResponse doesn't work everywhere, for example in `performAndValidateStartRequest` and in `handleStartFailed`